dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/openapi/Property.java

```diff
@@ -101,7 +110,19 @@ private static boolean isProperty(Field source) {
     return !isExcluded(source);
   }
 
-  private static boolean isProperty(Method source) {
+  private static boolean isSetter(Method source) {
+    String name = source.getName();
+    return !isExcluded(source)
+        && source.getParameterCount() == 1
+        && Stream.of("set")
```

Comment:
This seems unnecessarily complicated. 
Just `name.startsWith("set") && name.length() > 3 && isUpperCase(name.charAt(3))`  seems more clear.